dynamiq/connections/connections.py

```diff
@@ -932,6 +933,58 @@ def connect(self):
         pass
 
 
+class PGVector(BaseConnection):
```

Comment:
Should this be `PostgreSQL`?

Reply:
`pgvector` is just an extension for `PostgreSQL`

Reply:
I guess we can rename it to `PostgreSQL` if it makes sense.

Reply:
Let's rename it.

Reply:
Is it fine to leave everything else as `PGVector` and only rename the connection to `PostgreSQL`?

Reply:
Yeah, only the connection. The rest (nodes) are fine.

Reply:
Thanks, will do.

Reply:
Renamed the PGVector connection instances to PostgreSQL.